checker/build.xml

```diff
@@ -22,25 +22,20 @@
 
     <property environment="env"/>
 
+    <condition property="should-force-compile-java8">
```

Comment:
Could you add a comment for what the intuition for this variable is? It is confusing that it is checking for `jdk7orlower`, so it must mean something other than I would have expected.